package.json

```diff
@@ -6,14 +6,21 @@
   "module": "dist/esm/pushin.js",
   "main": "dist/umd/pushin.js",
   "typings": "dist/esm/index.d.ts",
+  "unpkg": "dist/umd/pushin.min.js",
```

Comment:
Is this for publishing to a CDN? I am curious to know more!

Reply:
The unpkg is already a CDN which gets files from npm. See this: https://cdn.cbd.int/#/.

So we even don't have to publish anything to CDN, the unpkg will already have it.

Reply:
No way! That is awesome! 💯